src/lib/holocene/accordion.svelte

```diff
@@ -1,6 +1,6 @@
 <script lang="ts">
-  import Icon from '$holocene/icon/icon.svelte';
-  import Badge from '$holocene/badge.svelte';
+  import Icon from '$lib/holocene/icon/icon.svelte';
```

Comment:
Have we looked into using [this](https://kit.svelte.dev/docs/configuration#alias) instead? Seems like it would create both the vite and tsconfig settings, but unsure if those settings are the issue here.

Reply:
Yeah we have alias config in both ui/cloud-ui.

Reply:
But in `vite.config.ts` and not [svelte.config.ts](https://kit.svelte.dev/docs/configuration#alias). Was wondering if there was something with the `tsconfig` that could be magically fixed by adding to `svelte.config.ts` instead.